relay-dynamic-config/src/defaults.rs

```diff
@@ -395,5 +400,51 @@ fn span_metrics(is_extract_all: bool) -> impl IntoIterator<Item = MetricSpec> {
                     .always(), // already guarded by condition on metric
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/webvital.score.total@ratio".into(),
+            field: Some("span.measurements.score.total.value".into()),
+            condition: Some(is_allowed_browser.clone()),
+            tags: vec![
+                Tag::with_key("transaction.op")
+                    .from_field("span.sentry_tags.transaction.op")
+                    .always(),
+                Tag::with_key("transaction")
+                    .from_field("span.sentry_tags.transaction")
+                    .always(),
+                Tag::with_key("environment")
+                    .from_field("span.sentry_tags.environment")
+                    .always(),
+                Tag::with_key("release")
+                    .from_field("span.sentry_tags.release")
+                    .always(),
+                Tag::with_key("browser.name")
+                    .from_field("span.browser.name")
+                    .always(), // already guarded by condition on metric
+            ],
+        },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/webvital.score.inp@ratio".into(),
+            field: Some("span.measurements.score.inp.value".into()),
+            condition: Some(is_allowed_browser),
```

Comment:
I would also add a test input to https://github.com/getsentry/relay/blob/f3ff94a1850be0b158250e454c6f650891c5931c/relay-server/src/metrics_extraction/event.rs (e.g. `test_extract_span_metrics`) to check if the condition actually works.